node/windows-packaging/CalicoWindows/kubernetes/kubelet-service.ps1

```diff
@@ -22,6 +22,8 @@ $baseDir = "$PSScriptRoot\.."
 ipmo $baseDir\libs\calico\calico.psm1 -Force
 
 Write-Host "Running kubelet service."
+Wait-ForCalicoInit
```

Comment:
I think it would be helpful to have a comment explaining why we're waiting for CalicoNode to finish initialiation

Reply:
If I correctly understand you I did it.